src/webrtc-wowza-player.ts

```diff
@@ -137,10 +137,19 @@ export class WowzaWebRTCPlayer extends EventEmitter {
       await pc.setRemoteDescription(sdpData);
 
       const description = await pc.createAnswer();
-      await pc.setLocalDescription(description);
+      const enhancer = new SDPEnhancer(this.videoConfigs, this.audioConfigs);
+      const upgradedDescription = this.sdpHandler
+        ? this.sdpHandler(
+            description,
+            sdp => enhancer.transformPlay(sdp),
+            'play'
+          )
+        : enhancer.transformPlay(description);
+
```

Comment:
With description being an awaited value, won't there be a possible issue of synchronisation ?